src/main/java/mate/academy/hibernate/relations/model/Movie.java

```diff
@@ -1,11 +1,28 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.JoinTable;
+import jakarta.persistence.ManyToMany;
+import jakarta.persistence.Table;
 import java.util.ArrayList;
 import java.util.List;
 
+@Entity
+@Table(name = "movies")
 public class Movie implements Cloneable {
```

Comment:
It is not recommended to make your entity classes `Cloneable`. This can lead to issues with Hibernate's management of the entity's lifecycle and may also cause problems with lazy loading. If you need to create a copy of an entity, consider using other patterns like Data Transfer Objects (DTO).